container-storage-setup.sh

```diff
@@ -49,6 +49,8 @@ _STATUSFILE_NAME="status"
 # Read man page for a description of currently supported options:
 # 'man container-storage-setup'
 
+# This variable used to be set by default in the config file
+_DEFAULT_CONTAINER_THINPOOL="docker-pool"
```

Comment:
This is used only in compatibility mode with docker. So will make sense to call it `_DEFAULT_DOCKER_THINPOOL`